CSS/ex_16/contact.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact | It's Just Jack</title>
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Homemade+Apple&display=swap');
+        header {
+            text-align: center ;
+            color:rgb(241, 77, 55) ;
+        }
+        h1 {
+            text-align: center;
+        }
+
+        h2 {
+            font-size: 30px ;
+            font-family: 'Homemade Apple', cursive ;
+        }
+        
+        /* Flexbox navigation goes below here */
+        .wrapper {
+            border-top: 5px dotted coral ;
+            border-bottom: 5px dotted coral ;
+            max-width: 200px ;
+            margin: auto ;
+            padding: 10px ;
+        }
+
+        .flex-nav ul {
+            list-style: none ;
+            margin: 0 ;
+            padding: 0 ;
+            display: flex ;
+        }
+
+        .flex-nav li {
+            flex: 1 ;
+        }
+
+        /*Inset border */
+        hr {
+            height: 10px ;
+	        border: 0 ;
+            box-shadow: 0 10px 10px -10px #8c8b8b inset ;
+            margin-top: 30px;
+        }
```

Comment:
It's also possible to achieve same effect styling <header> tag, adding border and box-shadow to it.
But what you did with <hr /> tag is nice and interesting.